src/app/layout/NavigationDrawer/NavigationDrawer.js

```diff
@@ -120,9 +131,53 @@ export default function TemporaryDrawer() {
     </div>
   );
 
+  const MissionTypeModal = () => (
```

Comment:
This should be in its own file.

Reply:
FWIW in terms of stories, there's a couple that relate to making a request without logging in. Well, its a bit more complicated but:

[As a recipient I can buy my own food boxes, and pay for it at that time, so a mission is created for those boxes and the mission funding status goes straight into ‘funded’ status ](https://airtable.com/tblRjd2McKgRW9MKj/viwAsqEsBiGxnqy7H/recNH5cqwqQmRIRZc?blocks=hide)

and

[As a recipient I can request a food box (to be funded by the organizer when the funds are available) so that I can receive food box because I need to eat and can't pay](https://airtable.com/tblRjd2McKgRW9MKj/viwAsqEsBiGxnqy7H/recGNl2T0LXh1ibbZ?blocks=hide)

This is part of the workflow that Karen L recently documented on Figma, but fwiw, also here:
![image](https://user-images.githubusercontent.com/166867/79186941-0ba42d80-7e6f-11ea-98c5-174676ba9f9f.png)

Karen also created an actual [clickable prototype](https://www.figma.com/proto/v5HdxDsvO2NzMuZU6CArzF/Resilience-App?node-id=1810%3A21&viewport=215%2C-40%2C0.6235744953155518&scaling=scale-down) of how it appears on the front page also (try this on mobile). 

Hope that wasn't too much info. Slightly overkill in response to your original question @qdozaq.. definitely not trying to be 'too much'... just super keen for everyone to be 100% on how this all ties together, which of course is only now being documented properly in design form...